packages/paste-customization/src/types/CreateCustomTheme.ts

```diff
@@ -5,6 +5,6 @@ import type {PasteCustomCSS} from './PasteCSS';
 export interface CreateCustomTheme {
   baseTheme: Partial<GenericThemeShape>;
   customBreakpoints?: string[];
-  elements?: PasteCustomCSS;
+  elements?: {[key: string]: PasteCustomCSS};
```

Comment:
Question: I wonder how this actually worked then, if I didn't type PasteCustomCSS as a keyed object?

Reply:
Unrelated, we need to make deepmerge a system library and share it. We bundle it in a number of components now